executor/aggregate_test.go

```diff
@@ -1373,3 +1374,63 @@ func (s *testSerialSuite) TestRandomPanicAggConsume(c *C) {
 		c.Assert(err.Error(), Equals, "failpoint panic: ERROR 1105 (HY000): Out Of Memory Quota![conn_id=1]")
 	}
 }
+
+func (s *testSuiteAgg) TestIssue15284(c *C) {
+	tk := testkit.NewTestKitWithInit(c, s.store)
```

Comment:
Add a test case for `isAllFirstRow is false`

Reply:
Done. PTAL